tests/system/Database/Live/ForgeTest.php

```diff
@@ -1475,4 +1475,219 @@ public function testDropPrimaryKey()
 
         $this->forge->dropTable('forge_test_users', true);
     }
+
+    public function testProcessIndexes()
```

Comment:
This is a bad test, too long. I don't understand what is tested.
Ideally one test method has one assertion.
Please split.

Reply:
Yes, when I created it it was testing a bunch more things. I have simplified it and commented it.